src/primitives/block.h

```diff
@@ -99,24 +103,54 @@ class CBlockHeader
 
     template <typename Stream, typename Operation>
     inline void SerializationOp(Stream& s, Operation ser_action) {
+        bool fCheckVeilDataHash = false;
         READWRITE(nVersion);
         READWRITE(hashPrevBlock);
-        READWRITE(hashVeilData);
+        if (nVersion >> 28 != 3) {
```

Comment:
See above.  Want to make sure we don't re-enable this when the block version goes up.

Reply:
Fixed